server/antibodyapi/hubmap/__init__.py

```diff
@@ -87,7 +85,7 @@ def hubmap_search():
         "target_symbol", "uniprot_accession_number", "clonality", "clone_id", "method",
         "tissue_preservation", "avr_pdf_filename", "antibody_hubmap_id",
 
-        "host", "cell_line", "cell_line_ontoloty_id",
```

Comment:
These two fields/columns should also be removed from the UI (result table and column selection) and CSV download. 
<img width="894" alt="Screenshot 2025-04-23 at 1 25 13 PM" src="https://github.com/user-attachments/assets/7e661554-d871-4540-952e-8de59f6d159f" />
<img width="1267" alt="Screenshot 2025-04-23 at 1 24 35 PM" src="https://github.com/user-attachments/assets/ad7972cb-178e-4e57-b30a-96229b4a9f07" />
<img width="1042" alt="Screenshot 2025-04-23 at 1 38 49 PM" src="https://github.com/user-attachments/assets/ebf692b2-d794-4b07-acf0-e2cf4be81e54" />